src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private final Random randomColor = new Random();
+
+    public String getRandomColor() {
+        return Color.values()[randomColor.nextInt(Color.values().length)].toString();
```

Comment:
Use the `name()` method instead of `toString()` for getting the string representation of enum constants to ensure that you always get the correct name of the enum constant.